plugins/action_modify_record/src/modifyRecord.ts

```diff
@@ -2,7 +2,15 @@ import { ActionQueueStatus } from '../../../src/generated/graphql'
 import { ActionPluginType } from '../../types'
 import databaseMethods, { DatabaseMethodsType } from './databaseMethods'
 import { DBConnectType } from '../../../src/components/databaseConnect'
-import { mapValues, get } from 'lodash'
+import { mapValues, get, snakeCase } from 'lodash'
+import { singular } from 'pluralize'
+
+// This will be prepended to NEW table created if not already present
+export const DATA_TABLE_PREFIX = 'data_table_'
+
+// These are the only tables in the system that we allow to be mutated with this
+// plugin. All other names will have "data_table_" prepended.
+const ALLOWED_TABLE_NAMES = ['user', 'organisation']
```

Comment:
This allows us to write to the "user" and "organisation" tables. But it also prevents writing to any other "core" table, which is a good thing. (Can add to this exception list later, if required)

Reply:
Should we also include `product` at this point? We were considering doing that. Just because is such a core thing, we could add the table with the minimum and add to this list to allow modifications

Reply:
> Should we also include `product` at this point? We were considering doing that. Just because is such a core thing, we could add the table with the minimum and add to this list to allow modifications

I think not. We haven't installed anywhere yet, so it's easy enough to update our current snapshots (we'll have to do it manually) to match this style. (Also, we'll need to add a record to the "data_table" table anyway, and that expects the prefix to NOT be in the name -- that's how sworup set it up.)

So I think it's better to just bite the bullet and change existing data tables (product, license, etc) before we get to far into installing somewhere.